### ISSUE TITLE
Forestplot: Dimensions are ordered from bottom to top.

### ISSUE THREAD

**Issue by aloctavodia** [arviz-devs/arviz#166]
See this example.

![index](https://user-images.githubusercontent.com/1338958/43984801-4cc959ee-9cd9-11e8-9cd7-b361ec1d226a.png)

**Comment by MFreidank** [comment]
Hey, I would be interested in doing a pr for this. 
It would be nice if you could provide the code you used to generate your example, so that I have a good starting point.

**Comment by aloctavodia** [comment]
Great!

You can try with a simpler example like 
```python
with pm.Model() as model:
    a = pm.Normal('a', 0, 1, shape=2)
    trace = pm.sample()
    
az.forestplot(trace)
```

**Comment by ColCarroll** [comment]
Oh, I did [a lot of reversing](https://github.com/arviz-devs/arviz/blob/master/arviz/plots/forestplot.py) here to get orders of things right, but failed to do so for `selection`.  The easiest solution might be to edit `xarray_var_iter` to add a `reverse_selections` flag to it.

**Comment by aloctavodia** [comment]
I say something wrong, the order of the variables is right, the problems is with multidimensional variables as the dimensions are ordered from bottom to top.

**Comment by ColCarroll** [comment]
Closed by #172
